### DISCUSSION TITLE
Please make a tutorial for non-systemd Linux OS (or better, to cater non-systemd Linux).

### DISCUSSION THREAD

**Discussion by NazcaLine** [DNSCrypt/dnscrypt-proxy#1987] [Ideas]
As usual, people on my Linux forum (Devuan) provides no help at all. Not suprising. They claim they could install it in their system but they can't explain how to do it. Instead, they refer me to how-to wiki for systemd.

I hope, you could do any modifications to dnscrypt-proxy to cater non-systemd Linux. But, it's OK if you can't or have time to do it. I still have Firefox with integrated DoH resolver.

**Comment by jedisct1** [comment]
What does the regular `dnscrypt-proxy -service install` command do on your system?

**Comment by ianbashford** [comment]
@NazcaLine  The service install command should likely work for SystemV, but I took a look at your distribution.

I ran up an LXC of Devuan; that image uses SystemV (I'm not sure what you use) and the dnscrypt-proxy package includes an upstart script `/etc/init.d/dnscrypt-proxy`   

You should be able to grab a copy of that, and use it even if you don't use the packaged version from Devuan (which is 2.0.19).  
 You can use that script, but have your own managed install and config files so you can stay on top of versions.
